From beefd1d27a8292a2fc12c5956400dedc0430e410 Mon Sep 17 00:00:00 2001
Message-Id: <beefd1d27a8292a2fc12c5956400dedc0430e410.1329747839.git.erack@redhat.com>
From: Eike Rathke <erack@redhat.com>
Date: Mon, 20 Feb 2012 15:12:40 +0100
Subject: [PATCH] resolved rhbz#794679 use proper Indian Rupee currency symbol
 U+20B9
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary="------------erAck-patch-parts"

This is a multi-part message in MIME format.
--------------erAck-patch-parts
Content-Type: text/plain; charset=UTF-8; format=fixed
Content-Transfer-Encoding: 8bit


Additionally fixes in gu-IN localization of day and month names contributed by
Ankitkumar Rameshchandra Patel <ankit@redhat.com>

Signed-off-by: Caolán McNamara <caolanm@redhat.com>
---
 i18npool/source/localedata/data/bn_IN.xml  |    9 ++-
 i18npool/source/localedata/data/en_IN.xml  |    9 ++-
 i18npool/source/localedata/data/gu_IN.xml  |  103 +++++++++++++++-------------
 i18npool/source/localedata/data/hi_IN.xml  |   16 ++++-
 i18npool/source/localedata/data/kn_IN.xml  |   16 ++++-
 i18npool/source/localedata/data/mai_IN.xml |    9 ++-
 i18npool/source/localedata/data/ml_IN.xml  |    9 ++-
 i18npool/source/localedata/data/mr_IN.xml  |    2 +-
 i18npool/source/localedata/data/or_IN.xml  |    9 ++-
 i18npool/source/localedata/data/pa_IN.xml  |    9 ++-
 i18npool/source/localedata/data/ta_IN.xml  |   16 ++++-
 i18npool/source/localedata/data/te_IN.xml  |   16 ++++-
 12 files changed, 164 insertions(+), 59 deletions(-)


--------------erAck-patch-parts
Content-Type: text/x-patch; name="0001-resolved-rhbz-794679-use-proper-Indian-Rupee-currenc.patch"
Content-Transfer-Encoding: 8bit
Content-Disposition: attachment; filename="0001-resolved-rhbz-794679-use-proper-Indian-Rupee-currenc.patch"

diff --git a/i18npool/source/localedata/data/bn_IN.xml b/i18npool/source/localedata/data/bn_IN.xml
index 1dc7992..cf845ea 100644
--- a/i18npool/source/localedata/data/bn_IN.xml
+++ b/i18npool/source/localedata/data/bn_IN.xml
@@ -34,7 +34,7 @@
     <TimePM>অপরাহ্ণ</TimePM>
     <MeasurementSystem>metric</MeasurementSystem>
   </LC_CTYPE>
-  <LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$৳-445]">
+  <LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$₹-445]">
     <FormatElement msgid="FixedFormatskey1" default="true" type="medium" usage="FIXED_NUMBER" formatindex="0">
       <FormatCode>Standard</FormatCode>
     </FormatElement>
@@ -308,6 +308,13 @@
   <LC_CURRENCY>
     <Currency default="true" usedInCompatibleFormatCodes="true">
       <CurrencyID>INR</CurrencyID>
+      <CurrencySymbol>₹</CurrencySymbol>
+      <BankSymbol>INR</BankSymbol>
+      <CurrencyName>Rupee</CurrencyName>
+      <DecimalPlaces>2</DecimalPlaces>
+    </Currency>
+    <Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+      <CurrencyID>INR</CurrencyID>
       <CurrencySymbol>৳</CurrencySymbol>
       <BankSymbol>INR</BankSymbol>
       <CurrencyName>Rupee</CurrencyName>
diff --git a/i18npool/source/localedata/data/en_IN.xml b/i18npool/source/localedata/data/en_IN.xml
index 9c404f5..c802f64 100644
--- a/i18npool/source/localedata/data/en_IN.xml
+++ b/i18npool/source/localedata/data/en_IN.xml
@@ -12,7 +12,7 @@
     </Country>
   </LC_INFO>
   <LC_CTYPE ref="en_GB"/>
-  <LC_FORMAT ref="en_GB" replaceTo="[$Rs.-4009]"/>
+  <LC_FORMAT ref="en_GB" replaceTo="[$₹-4009]"/>
   <LC_COLLATION ref="en_US"/>
   <LC_SEARCH ref="en_US"/>
   <LC_INDEX ref="en_US"/>
@@ -30,6 +30,13 @@
   <LC_CURRENCY>
     <Currency default="true" usedInCompatibleFormatCodes="true">
       <CurrencyID>INR</CurrencyID>
+      <CurrencySymbol>₹</CurrencySymbol>
+      <BankSymbol>INR</BankSymbol>
+      <CurrencyName>Rupee</CurrencyName>
+      <DecimalPlaces>2</DecimalPlaces>
+    </Currency>
+    <Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+      <CurrencyID>INR</CurrencyID>
       <CurrencySymbol>Rs.</CurrencySymbol>
       <BankSymbol>INR</BankSymbol>
       <CurrencyName>Indian Rupee</CurrencyName>
diff --git a/i18npool/source/localedata/data/gu_IN.xml b/i18npool/source/localedata/data/gu_IN.xml
index f5923a7..6097289 100644
--- a/i18npool/source/localedata/data/gu_IN.xml
+++ b/i18npool/source/localedata/data/gu_IN.xml
@@ -12,7 +12,7 @@
 </Country>
 </LC_INFO>
 <LC_CTYPE ref="hi_IN"/>
-<LC_FORMAT ref="hi_IN" replaceTo="[$ઋુ.-447]"/>
+<LC_FORMAT ref="hi_IN" replaceTo="[$₹-447]"/>
 <LC_COLLATION ref="hi_IN"/>
 <LC_SEARCH ref="hi_IN"/>
 <LC_INDEX ref="en_US"/>
@@ -21,112 +21,112 @@
 <DaysOfWeek>
 <Day>
 <DayID>sun</DayID>
-<DefaultAbbrvName>ઋવિ.</DefaultAbbrvName>
-<DefaultFullName>ઋવિવાઋ</DefaultFullName>
+<DefaultAbbrvName>રવિ</DefaultAbbrvName>
+<DefaultFullName>રવિવાર</DefaultFullName>
 </Day>
 <Day>
 <DayID>mon</DayID>
-<DefaultAbbrvName>સોમ.</DefaultAbbrvName>
-<DefaultFullName>સોમવાઋ્</DefaultFullName>
+<DefaultAbbrvName>સોમવાર</DefaultAbbrvName>
+<DefaultFullName>સોમવાર</DefaultFullName>
 </Day>
 <Day>
 <DayID>tue</DayID>
-<DefaultAbbrvName>મંગલ.</DefaultAbbrvName>
-<DefaultFullName>મંગલવાઋ</DefaultFullName>
+<DefaultAbbrvName>મંગળ</DefaultAbbrvName>
+<DefaultFullName>મંગળવાર</DefaultFullName>
 </Day>
 <Day>
 <DayID>wed</DayID>
-<DefaultAbbrvName>બુધ.</DefaultAbbrvName>
-<DefaultFullName>બુધવાઋ</DefaultFullName>
+<DefaultAbbrvName>બુધ</DefaultAbbrvName>
+<DefaultFullName>બુધવાર</DefaultFullName>
 </Day>
 <Day>
 <DayID>thu</DayID>
-<DefaultAbbrvName>ગુઋુ.</DefaultAbbrvName>
-<DefaultFullName>ગુઋુવાઋ</DefaultFullName>
+<DefaultAbbrvName>ગુરુ</DefaultAbbrvName>
+<DefaultFullName>ગુરુવાર</DefaultFullName>
 </Day>
 <Day>
 <DayID>fri</DayID>
-<DefaultAbbrvName>શુક્ઋ.</DefaultAbbrvName>
-<DefaultFullName>શુક્ઋવાઋ</DefaultFullName>
+<DefaultAbbrvName>શુક્ર</DefaultAbbrvName>
+<DefaultFullName>શુક્રવાર</DefaultFullName>
 </Day>
 <Day>
 <DayID>sat</DayID>
-<DefaultAbbrvName>શનિ.</DefaultAbbrvName>
-<DefaultFullName>શનિવાઋ</DefaultFullName>
+<DefaultAbbrvName>શનિ</DefaultAbbrvName>
+<DefaultFullName>શનિવાર</DefaultFullName>
 </Day>
 </DaysOfWeek>
 <MonthsOfYear>
 <Month>
 <MonthID>jan</MonthID>
-<DefaultAbbrvName>જનવઋી.</DefaultAbbrvName>
-<DefaultFullName>જનવઋી</DefaultFullName>
+<DefaultAbbrvName>જાન</DefaultAbbrvName>
+<DefaultFullName>જાન્યુઆરી</DefaultFullName>
 </Month>
 <Month>
 <MonthID>feb</MonthID>
-<DefaultAbbrvName>ફઋવઋી.</DefaultAbbrvName>
-<DefaultFullName>ફઋવઋી</DefaultFullName>
+<DefaultAbbrvName>ફેબ</DefaultAbbrvName>
+<DefaultFullName>ફેબ્રુઆરી</DefaultFullName>
 </Month>
 <Month>
 <MonthID>mar</MonthID>
-<DefaultAbbrvName>માઋ્ચ.</DefaultAbbrvName>
-<DefaultFullName>માઋ્ચ</DefaultFullName>
+<DefaultAbbrvName>માર્ચ</DefaultAbbrvName>
+<DefaultFullName>માર્ચ</DefaultFullName>
 </Month>
 <Month>
 <MonthID>apr</MonthID>
-<DefaultAbbrvName>અપ્ઋેલ.</DefaultAbbrvName>
-<DefaultFullName>અપ્ઋેલ</DefaultFullName>
+<DefaultAbbrvName>એપ્રિલ</DefaultAbbrvName>
+<DefaultFullName>એપ્રિલ</DefaultFullName>
 </Month>
 <Month>
 <MonthID>may</MonthID>
-<DefaultAbbrvName>મઈ.</DefaultAbbrvName>
-<DefaultFullName>મઈ</DefaultFullName>
+<DefaultAbbrvName>મે</DefaultAbbrvName>
+<DefaultFullName>મે</DefaultFullName>
 </Month>
 <Month>
 <MonthID>jun</MonthID>
-<DefaultAbbrvName>જૂન.</DefaultAbbrvName>
+<DefaultAbbrvName>જૂન</DefaultAbbrvName>
 <DefaultFullName>જૂન</DefaultFullName>
 </Month>
 <Month>
 <MonthID>jul</MonthID>
-<DefaultAbbrvName>જુલાઇ.</DefaultAbbrvName>
+<DefaultAbbrvName>જુલ</DefaultAbbrvName>
 <DefaultFullName>જુલાઇ</DefaultFullName>
 </Month>
 <Month>
 <MonthID>aug</MonthID>
-<DefaultAbbrvName>અગસ્ત.</DefaultAbbrvName>
-<DefaultFullName>અગસ્ત</DefaultFullName>
+<DefaultAbbrvName>ઑગ</DefaultAbbrvName>
+<DefaultFullName>ઑગસ્ટ</DefaultFullName>
 </Month>
 <Month>
 <MonthID>sep</MonthID>
-<DefaultAbbrvName>સિતંબઋ.</DefaultAbbrvName>
-<DefaultFullName>સિતંબઋ</DefaultFullName>
+<DefaultAbbrvName>સપ્ટ</DefaultAbbrvName>
+<DefaultFullName>સપ્ટેમ્બર</DefaultFullName>
 </Month>
 <Month>
 <MonthID>oct</MonthID>
-<DefaultAbbrvName>અક્તોબઋ.</DefaultAbbrvName>
-<DefaultFullName>અક્તોબઋ</DefaultFullName>
+<DefaultAbbrvName>ઑક્ટ</DefaultAbbrvName>
+<DefaultFullName>ઑક્ટોબર</DefaultFullName>
 </Month>
 <Month>
 <MonthID>nov</MonthID>
-<DefaultAbbrvName>નવંબઋ.</DefaultAbbrvName>
-<DefaultFullName>નવંબઋ</DefaultFullName>
+<DefaultAbbrvName>નવ</DefaultAbbrvName>
+<DefaultFullName>નવેમ્બર</DefaultFullName>
 </Month>
 <Month>
 <MonthID>dec</MonthID>
-<DefaultAbbrvName>ડિસધંબઋ.</DefaultAbbrvName>
-<DefaultFullName>ડિસંબઋ</DefaultFullName>
+<DefaultAbbrvName>ડિસ</DefaultAbbrvName>
+<DefaultFullName>ડિસેમ્બર</DefaultFullName>
 </Month>
 </MonthsOfYear>
 <Eras>
 <Era>
 <EraID>bc</EraID>
-<DefaultAbbrvName>ઈસાપૂઋ્વ.હ્ન</DefaultAbbrvName>
-<DefaultFullName>ઈસાપૂઋ્વ.</DefaultFullName>
+<DefaultAbbrvName>ઈ.સ.પૂ.</DefaultAbbrvName>
+<DefaultFullName>ઈસવીસનપૂર્વે</DefaultFullName>
 </Era>
 <Era>
 <EraID>ad</EraID>
-<DefaultAbbrvName>સન.</DefaultAbbrvName>
-<DefaultFullName>સન.</DefaultFullName>
+<DefaultAbbrvName>ઈ.સ.</DefaultAbbrvName>
+<DefaultFullName>ઈસવીસન</DefaultFullName>
 </Era>
 </Eras>
 <StartDayOfWeek>
@@ -138,6 +138,13 @@
 <LC_CURRENCY>
 <Currency default="true" usedInCompatibleFormatCodes="true">
 <CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
 <CurrencySymbol>ઋુ.</CurrencySymbol>
 <BankSymbol>INR</BankSymbol>
 <CurrencyName>Rupee</CurrencyName>
@@ -154,13 +161,13 @@
 <LineMode/>
 </BreakIteratorRules>
 <ReservedWords>
-<trueWord>સહી</trueWord>
-<falseWord>ગલત</falseWord>
-<quarter1Word>પેહલા હંત 1</quarter1Word>
-<quarter2Word>ડૂસઋા હંત 2</quarter2Word>
-<quarter3Word>તીસઋા હંત 3</quarter3Word>
-<quarter4Word>ચૌતા હંત 4</quarter4Word>
-<aboveWord>ઊપઋ</aboveWord>
+<trueWord>ખરું</trueWord>
+<falseWord>ખોટું</falseWord>
+<quarter1Word>ચતુર્થાંશ૧</quarter1Word>
+<quarter2Word>ચતુર્થાંશ૨</quarter2Word>
+<quarter3Word>ચતુર્થાંશ૩</quarter3Word>
+<quarter4Word>ચતુર્થાંશ૪</quarter4Word>
+<aboveWord>ઊપર</aboveWord>
 <belowWord>નીચે</belowWord>
 <quarter1Abbreviation>Q1</quarter1Abbreviation>
 <quarter2Abbreviation>Q2</quarter2Abbreviation>
diff --git a/i18npool/source/localedata/data/hi_IN.xml b/i18npool/source/localedata/data/hi_IN.xml
index cfca665..220cfc6 100644
--- a/i18npool/source/localedata/data/hi_IN.xml
+++ b/i18npool/source/localedata/data/hi_IN.xml
@@ -34,7 +34,7 @@
 <TimePM>PM</TimePM>
 <MeasurementSystem>Metric</MeasurementSystem>
 </LC_CTYPE>
-<LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$रु.-439]">
+<LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$₹-439]">
 <FormatElement msgid="DateFormatskey1" default="true" type="short" usage="DATE" formatindex="18">
 <FormatCode>D-M-YY</FormatCode>
 <DefaultName></DefaultName>
@@ -350,6 +350,20 @@
 <LC_CURRENCY>
 <Currency default="true" usedInCompatibleFormatCodes="true">
 <CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
 <CurrencySymbol>रु.</CurrencySymbol>
 <BankSymbol>INR</BankSymbol>
 <CurrencyName>Rupee</CurrencyName>
diff --git a/i18npool/source/localedata/data/kn_IN.xml b/i18npool/source/localedata/data/kn_IN.xml
index 00d5dcf..7c3d3de 100644
--- a/i18npool/source/localedata/data/kn_IN.xml
+++ b/i18npool/source/localedata/data/kn_IN.xml
@@ -12,7 +12,7 @@
 </Country>
 </LC_INFO>
 <LC_CTYPE ref="hi_IN"/>
-<LC_FORMAT ref="hi_IN" replaceTo="[$ರೂ.-44b]"/>
+<LC_FORMAT ref="hi_IN" replaceTo="[$₹-44b]"/>
 <LC_COLLATION ref="en_US"/>
 <LC_SEARCH ref="en_US"/>
 <LC_INDEX ref="en_US"/>
@@ -138,6 +138,20 @@
 <LC_CURRENCY>
 <Currency default="true" usedInCompatibleFormatCodes="true">
 <CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
 <CurrencySymbol>ರೂ.</CurrencySymbol>
 <BankSymbol>INR</BankSymbol>
 <CurrencyName>Rupee</CurrencyName>
diff --git a/i18npool/source/localedata/data/mai_IN.xml b/i18npool/source/localedata/data/mai_IN.xml
index 07c3d39..ee9fe69 100644
--- a/i18npool/source/localedata/data/mai_IN.xml
+++ b/i18npool/source/localedata/data/mai_IN.xml
@@ -34,7 +34,7 @@
     <TimePM>अपराह्न</TimePM>
     <MeasurementSystem>metric</MeasurementSystem>
   </LC_CTYPE>
-  <LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$रू.-645]">
+  <LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$₹-645]">
     <FormatElement msgid="FixedFormatskey1" default="true" type="medium" usage="FIXED_NUMBER" formatindex="0">
       <FormatCode>General</FormatCode>
     </FormatElement>
@@ -315,6 +315,13 @@
   <LC_CURRENCY>
     <Currency default="true" usedInCompatibleFormatCodes="true">
       <CurrencyID>INR</CurrencyID>
+      <CurrencySymbol>₹</CurrencySymbol>
+      <BankSymbol>INR</BankSymbol>
+      <CurrencyName>Rupee</CurrencyName>
+      <DecimalPlaces>2</DecimalPlaces>
+    </Currency>
+    <Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+      <CurrencyID>INR</CurrencyID>
       <CurrencySymbol>रू.</CurrencySymbol>
       <BankSymbol>INR</BankSymbol>
       <CurrencyName>रुपया</CurrencyName>
diff --git a/i18npool/source/localedata/data/ml_IN.xml b/i18npool/source/localedata/data/ml_IN.xml
index 813e523..4adb8d4 100644
--- a/i18npool/source/localedata/data/ml_IN.xml
+++ b/i18npool/source/localedata/data/ml_IN.xml
@@ -34,7 +34,7 @@
     <TimePM>PM</TimePM>
     <MeasurementSystem>metric</MeasurementSystem>
   </LC_CTYPE>
-  <LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$ക.-44C]">
+  <LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$₹-44C]">
     <FormatElement msgid="FixedFormatskey1" default="true" type="medium" usage="FIXED_NUMBER" formatindex="0">
       <FormatCode>Standard</FormatCode>
     </FormatElement>
@@ -306,6 +306,13 @@
   <LC_CURRENCY>
     <Currency default="true" usedInCompatibleFormatCodes="true">
       <CurrencyID>INR</CurrencyID>
+      <CurrencySymbol>₹</CurrencySymbol>
+      <BankSymbol>INR</BankSymbol>
+      <CurrencyName>Rupee</CurrencyName>
+      <DecimalPlaces>2</DecimalPlaces>
+    </Currency>
+    <Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+      <CurrencyID>INR</CurrencyID>
       <CurrencySymbol>ക.</CurrencySymbol>
       <BankSymbol>INR</BankSymbol>
       <CurrencyName>Rupee</CurrencyName>
diff --git a/i18npool/source/localedata/data/mr_IN.xml b/i18npool/source/localedata/data/mr_IN.xml
index efc238f..84aa6f9 100644
--- a/i18npool/source/localedata/data/mr_IN.xml
+++ b/i18npool/source/localedata/data/mr_IN.xml
@@ -12,7 +12,7 @@
 </Country>
 </LC_INFO>
 <LC_CTYPE ref="hi_IN"/>
-<LC_FORMAT ref="hi_IN" replaceTo="[$रु.-44e]"/>
+<LC_FORMAT ref="hi_IN" replaceTo="[$₹-44E]"/>
 <LC_COLLATION ref="en_US"/>
 <LC_SEARCH ref="en_US"/>
 <LC_INDEX ref="en_US"/>
diff --git a/i18npool/source/localedata/data/or_IN.xml b/i18npool/source/localedata/data/or_IN.xml
index 2233345..b2f6415 100644
--- a/i18npool/source/localedata/data/or_IN.xml
+++ b/i18npool/source/localedata/data/or_IN.xml
@@ -34,7 +34,7 @@
     <TimePM>ଅପରାହ୍ନ</TimePM>
     <MeasurementSystem>metric</MeasurementSystem>
   </LC_CTYPE>
-  <LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$ଟ-448]">
+  <LC_FORMAT replaceFrom="[CURRENCY]" replaceTo="[$₹-448]">
     <FormatElement msgid="FixedFormatskey1" default="true" type="medium" usage="FIXED_NUMBER" formatindex="0">
       <FormatCode>General</FormatCode>
     </FormatElement>
@@ -315,6 +315,13 @@
   <LC_CURRENCY>
     <Currency default="true" usedInCompatibleFormatCodes="true">
       <CurrencyID>INR</CurrencyID>
+      <CurrencySymbol>₹</CurrencySymbol>
+      <BankSymbol>INR</BankSymbol>
+      <CurrencyName>Rupee</CurrencyName>
+      <DecimalPlaces>2</DecimalPlaces>
+    </Currency>
+    <Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+      <CurrencyID>INR</CurrencyID>
       <CurrencySymbol>ଟ</CurrencySymbol>
       <BankSymbol>INR</BankSymbol>
       <CurrencyName>Indian Rupee</CurrencyName>
diff --git a/i18npool/source/localedata/data/pa_IN.xml b/i18npool/source/localedata/data/pa_IN.xml
index 8ca9444..a414fd9 100644
--- a/i18npool/source/localedata/data/pa_IN.xml
+++ b/i18npool/source/localedata/data/pa_IN.xml
@@ -34,7 +34,7 @@
     <TimePM>ਸ਼ਾਮ</TimePM>
     <MeasurementSystem>metric</MeasurementSystem>
   </LC_CTYPE>
-  <LC_FORMAT ref="hi_IN" replaceTo="[$ਰੁ.-446]"/>
+  <LC_FORMAT ref="hi_IN" replaceTo="[$₹-446]"/>
   <LC_COLLATION ref="hi_IN" />
   <LC_SEARCH ref="hi_IN"/>
   <LC_INDEX ref="en_US"/>
@@ -160,6 +160,13 @@
   <LC_CURRENCY>
     <Currency default="true" usedInCompatibleFormatCodes="true">
       <CurrencyID>INR</CurrencyID>
+      <CurrencySymbol>₹</CurrencySymbol>
+      <BankSymbol>INR</BankSymbol>
+      <CurrencyName>Rupee</CurrencyName>
+      <DecimalPlaces>2</DecimalPlaces>
+    </Currency>
+    <Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+      <CurrencyID>INR</CurrencyID>
       <CurrencySymbol>ਰੁ.</CurrencySymbol>
       <BankSymbol>INR</BankSymbol>
       <CurrencyName>Rupee</CurrencyName>
diff --git a/i18npool/source/localedata/data/ta_IN.xml b/i18npool/source/localedata/data/ta_IN.xml
index a06e374..8cbb5f6 100644
--- a/i18npool/source/localedata/data/ta_IN.xml
+++ b/i18npool/source/localedata/data/ta_IN.xml
@@ -12,7 +12,7 @@
 </Country>
 </LC_INFO>
 <LC_CTYPE ref="hi_IN"/>
-<LC_FORMAT ref="hi_IN" replaceTo="[$ரூ.-449]"/>
+<LC_FORMAT ref="hi_IN" replaceTo="[$₹-449]"/>
 <LC_COLLATION ref="en_US"/>
 <LC_SEARCH ref="en_US"/>
 <LC_INDEX ref="en_US"/>
@@ -138,6 +138,20 @@
 <LC_CURRENCY>
 <Currency default="true" usedInCompatibleFormatCodes="true">
 <CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
 <CurrencySymbol>ரூ.</CurrencySymbol>
 <BankSymbol>INR</BankSymbol>
 <CurrencyName>Rupee</CurrencyName>
diff --git a/i18npool/source/localedata/data/te_IN.xml b/i18npool/source/localedata/data/te_IN.xml
index b753b34..d084fda 100644
--- a/i18npool/source/localedata/data/te_IN.xml
+++ b/i18npool/source/localedata/data/te_IN.xml
@@ -12,7 +12,7 @@
 </Country>
 </LC_INFO>
 <LC_CTYPE ref="hi_IN"/>
-<LC_FORMAT ref="hi_IN" replaceTo="[$రృ.-44a]"/>
+<LC_FORMAT ref="hi_IN" replaceTo="[$₹-44a]"/>
 <LC_COLLATION ref="en_US"/>
 <LC_SEARCH ref="en_US"/>
 <LC_INDEX ref="en_US"/>
@@ -137,6 +137,20 @@
 <LC_CURRENCY>
 <Currency default="true" usedInCompatibleFormatCodes="true">
 <CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
+<CurrencySymbol>₹</CurrencySymbol>
+<BankSymbol>INR</BankSymbol>
+<CurrencyName>Rupee</CurrencyName>
+<DecimalPlaces>2</DecimalPlaces>
+</Currency>
+<Currency default="false" usedInCompatibleFormatCodes="false" legacyOnly="true">
+<CurrencyID>INR</CurrencyID>
 <CurrencySymbol>రృ.</CurrencySymbol>
 <BankSymbol>INR</BankSymbol>
 <CurrencyName>Rupee</CurrencyName>

--------------erAck-patch-parts--


